### DISCUSSION TITLE
Flag to disable rotating comments?

### DISCUSSION THREAD

**Discussion by RhapsodySL** [PoomSmart/YTABConfig#107] [Q&A]
About every minute, the Comments section preview under a video (on iPad view, unsure if this happens on iPhone view) will flash and display a new comment. I find this somewhat distracting and irritating, so I’m wondering if there’s a flag to disable it (or some other way).

**Comment by hwang1607** [comment]
Not sure if this helps but in YTliteplus there is an option to disable comment previews under player

**Reply by DeerSpotter** [reply]
I am actually looking for a
Way to go to comment when you press on it.
